elide-model-config/README.md

```diff
@@ -6,8 +6,10 @@ To build and run:
 ```text
 1. mvn clean install
 2. Execute Jar File :
-   a) java -cp elide-contrib/elide-dynamic-config-helpers/target/elide-dynamic-config-*-jar-with-dependencies.jar com.yahoo.elide.contrib.dynamicconfighelpers.validator.DynamicConfigValidator --configDir <Path for Config Directory>
-   b) java -cp elide-contrib/elide-dynamic-config-helpers/target/elide-dynamic-config-*-jar-with-dependencies.jar com.yahoo.elide.contrib.dynamicconfighelpers.validator.DynamicConfigValidator --help
+   a) java -cp elide-spring/elide-spring-boot-autoconfigure/target/elide-spring-boot-autoconfigure-*.jar com.yahoo.elide.contrib.dynamicconfighelpers.validator.DynamicConfigValidator --configDir <Path for Config Directory>
+      java -cp elide-standalone/target/elide-standalone-*.jar com.yahoo.elide.contrib.dynamicconfighelpers.validator.DynamicConfigValidator --configDir <Path for Config Directory>
```

Comment:
Let's not include the path to the jar - and we don't need to include instructions for both spring and standalone.  Let's make the instructions generic:
1. Build your app
2. run the validator: ...